MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/Reflectometry/IReflSaveTabView.h

```diff
@@ -51,7 +55,6 @@ class DLLExport IReflSaveTabView {
   virtual std::string getCurrentWorkspaceName() const = 0;
   virtual std::vector<std::string> getSelectedWorkspaces() const = 0;
   virtual std::vector<std::string> getSelectedParameters() const = 0;
-  virtual std::string getSpectraList() const = 0;
```

Comment:
I see this has been removed, but the text box widget is still visible, do we need it?